<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmyclass</class>
 <widget class="QDialog" name="addmyclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="classlabel">
        <property name="text">
         <string>课程名称</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>122</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="nametextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="classmsglabel">
        <property name="text">
         <string>课程信息</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="classmsgtextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="finalcheckBox">
        <property name="text">
         <string>是否有期末</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="finallabel">
        <property name="text">
         <string>期末时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="monthspinBox">
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="monthlabel">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="dayspinBox">
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="daylabel">
        <property name="text">
         <string>日</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="hourspinBox">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hourlabel">
        <property name="text">
         <string>点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="mintuespinBox">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mintuelabel">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addmyclass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addmyclass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
